cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/test/java/io/cdap/cdap/etl/spark/batch/relation/SparkSQLRelationTest.java

```diff
@@ -83,9 +84,9 @@ public void testFilter() {
     SparkSQLRelation actualRelation = (SparkSQLRelation) baseSQLRelation.filter(factory.compile("a > 2"));
     SparkSQLRelation expectedRelation = new SparkSQLRelation("testDataset",
                                                              Arrays.asList("a", "b"),
-                                                             "SELECT a AS a , b AS b FROM testDataset " +
+                                                             "SELECT a AS a , b AS b FROM testDataset AS testDataset " +
                                                                "WHERE a > 2",
-                                                             null);
+                                                             null, null);
```

Comment:
Can we add some tests with multiple chained relations? all the tests are for single relation.

Reply:
Added